### PR TITLE
Update credential-guard-manage.md

### PR DESCRIPTION
**Pull request by MSFTandrelom**
Customer pointed out that while they were following steps outlined in the article, they ran into some errors. Validation revealed that step 3 in the feature installation procedure is outdated as of Windows 10 1607. Added a note about that

### CODE REVIEW

File: windows/security/identity-protection/credential-guard/credential-guard-manage.md [hunk 1]
```diff
@@ -86,6 +86,8 @@ You can do this by using either the Control Panel or the Deployment Image Servic
     ```
     dism /image:<WIM file name> /Enable-Feature /FeatureName:IsolatedUserMode
     ```
+> [!NOTE]
+> In Windows 10 1607 or later versions, Isolated User Mode feature has been integrated into the core Operating system, so running the above command step 3 is no longer necessary
```
Comment (illfated) [inline windows/security/identity-protection/credential-guard/credential-guard-manage.md]:
```suggestion
> In Windows 10 1607 and later versions, the Isolated User Mode feature has been integrated into the core operating system. Running the command in step 3 above is therefore no longer necessary.
```
(Either "Operating System" or "operating system", not capitalized for only one of them (except for at the start of a sentence, of course).)

File: windows/security/identity-protection/credential-guard/credential-guard-manage.md [hunk 2]
```diff
@@ -86,6 +86,8 @@ You can do this by using either the Control Panel or the Deployment Image Servic
     ```
     dism /image:<WIM file name> /Enable-Feature /FeatureName:IsolatedUserMode
     ```
+> [!NOTE]
+> In Windows 10 1607 and later versions, the Isolated User Mode feature has been integrated into the core operating system. Running the command in step 3 above is therefore no longer required.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/credential-guard/credential-guard-manage.md]:
```suggestion
> In Windows 10, version 1607 and later versions, the Isolated User Mode feature has been integrated into the core operating system. Running the command in step 3 above is therefore no longer required.
```

Comment (illfated) [reply windows/security/identity-protection/credential-guard/credential-guard-manage.md]: Suggestion for your suggestion:
```suggestion
> In Windows 10, version 1607 and later, the Isolated User Mode feature has been integrated into the core operating system. Running the command in step 3 above is therefore no longer required.
```

**Comment by e0i** [comment]
@DulceMontemayor The PR that introduces a clarifying note to an article Windows Defender Credential Guard that is created by a MS contributor is available for review. Thanks.

**Comment by e0i** [comment]
@DulceMontemayor Following up for updates.

cc @Dansimp

**Comment by e0i** [comment]
@Dansimp The PR that has been approved by the document author can be merged. Thanks.

**Comment by illfated** [comment]
@JohanFreelancer9 : Just a minor thought, would it be useful to highlight the keywords in the note in **bold**, or is it enough that they are capitalized already?
example:
> In Windows 10, version 1607 and later, the **Isolated User Mode** feature has been integrated into the core operating system. Running the command in step 3 above is therefore no longer required.

(For all I know, the focus may be clear enough already. I just thought it would be appropriate to ask.)

**Comment by JohanFreelancer9** [comment]
@illfated Good question, but I would be very happy to leave it as it is, as the note is purely informative.

**Comment by illfated** [comment]
That is as good an answer as any. I concur with your assessment.

**Comment by e0i** [comment]
Still under escalation.

**Comment by e0i** [comment]
@Dansimp The PR that has been approved by the document author can be merged. Thanks.

**Comment by e0i** [comment]
@MSFTandrelom We'd like to merge your contributions. Would you change the PR merge target from `master` to `public`? Thanks.

**Pull request merged**